Templates/AppSource App/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -102,7 +103,6 @@ jobs:
             parentTelemetryScopeJson: ${{ inputs.parentTelemetryScopeJson }}
             project: ${{ inputs.project }}
             settingsJson: ${{ env.Settings }}
-            secretsJson: ${{ env.RepoSecrets }}
```

Comment:
Isn't this breaking? 
In the case of pull_request_target trigger, _DetermineArtifactUrl_ needs the secrets to determine the artifact URL.